### DISCUSSION TITLE
gptstudio highlighted in rweekly 🥇

### DISCUSSION THREAD

**Discussion by calderonsamuel** [MichelNivard/gptstudio#108] [Show and tell]
Good news!

Last week, I made a blog post about some challenges presented in the development of GPTStudio v0.2.0. You can find it here: [https://samuelenrique.com/posts/2023-06-02-updating-gptstudio/](https://samuelenrique.com/posts/2023-06-02-updating-gptstudio/)

I submitted it to rweekly, and it looks like it has been featured as a highlight of the week! [https://rweekly.org/2023-W23.html](https://rweekly.org/2023-W23.html)

I believe this means that it will be discussed in the rweekly podcast. This link will work once it gets uploaded: [https://rweekly.fireside.fm/125](https://rweekly.fireside.fm/125)

I hope this attracts new users and collaborators.

**Comment by JamesHWade** [comment]
Woot woot! Wonderful blog post. Thank you for writing that and submitting to rweekly!!!
